### ISSUE TITLE
Regular expression causes SyntaxError after compilation

### ISSUE THREAD

**Issue by tdjsnelling** [zeit/pkg#179]
When I run my .exe on Windows, I get the following error:
```
C:\Users\Tom Snelling\Documents\FILES\CODE\PROJECTS\multicop>pkg package.json -t win
> pkg@4.1.3
> Warning Cannot resolve 'path.join(parentDir, opts.pkg)'
  C:\Users\Tom Snelling\Documents\FILES\CODE\PROJECTS\multicop\node_modules\meow\index.js
  Dynamic require may fail at run time, because the requested file
  is unknown at compilation time and not included into executable.
  Use a string literal as an argument for 'require', or leave it
  as is and specify the resolved file name in 'scripts' option.

C:\Users\Tom Snelling\Documents\FILES\CODE\PROJECTS\multicop>multicop.exe
pkg/prelude/bootstrap.js:1102
      throw error;
      ^

SyntaxError: Unexpected identifier
    at Function.compile (C:\snapshot\multicop\node_modules\minstache\index.js:0)
    at Object.<anonymous> (C:\snapshot\multicop\node_modules\nightmare\lib\javascript.js:73:29)
    at Module._compile (pkg/prelude/bootstrap.js:1173:22)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at Module.require (pkg/prelude/bootstrap.js:1083:31)
    at require (internal/module.js:20:19)
```

**Comment by tdjsnelling** [comment]
I have also tried compiling with `--public` but from what I understand this only covers top-level code - so I am unable to find the line in `minstache/index.js` that is causing a problem after compilation.

**Comment by tdjsnelling** [comment]
Through commenting lines out and deduction, I have determined that the problematic lines are [these ones](https://github.com/visionmedia/minstache/blob/master/index.js#L74-L77).

```
js = '\n'
    + indent(escape.toString()) + ';\n\n'
    + indent(section.toString()) + ';\n\n'
    + '  return ' + js.join('').replace(/\n/g, '\\n');
```

I am unsure what about this code causes the SyntaxError after compilation, but when commented out and compiled again my .exe will run - but then of course crash because I have removed some of the necessary minstache code.

**Comment by igorklopov** [comment]
It does not entirely solve your issue, but you will spot the remaining problem easily (lack of `node_modules/sliced` etc) and organize your deployment directory accordingly. https://github.com/zeit/pkg/tree/master/test/test-79-npm/nightmare

**Comment by igorklopov** [comment]
Your minstache problem has another solution. https://github.com/zeit/pkg/blob/master/test/test-79-npm/nightmare/nightmare.js#L12. `evaluate` looks like function source is sent to electron. It means the file, that contains it, should be covered by `--public` or placed to assets.

**Comment by tdjsnelling** [comment]
Brilliant, everything seems to be in working order now. Cheers

**Comment by igorklopov** [comment]
Fixed in pkg@4.2.0. Please upgrade!
